addons/mail/static/src/models/composer/composer.js

```diff
@@ -510,6 +594,9 @@ function factory(dependencies) {
             ],
             default: false,
         }),
+        delimiter: attr({
+            default: "",
+        }),
```

Comment:
need better name + doc, before looking at the rest of the code I have absolutely no idea what this it

Reply:
Done